homework/assignment_5/baseball_features.sql

```diff
@@ -0,0 +1,206 @@
+    END AS home_hrh,
+    CASE
+        WHEN tbc.Strikeout = 0 THEN 0
+        ELSE tbc.Walk/tbc.Strikeout
+    END AS home_bb_k
+FROM
+    game g
+JOIN team_batting_counts tbc
+ON tbc.team_id = g.home_team_id
+GROUP BY
+    g.game_id
+ORDER BY
+    g.game_id, g.home_team_id DESC;
+
+#####################################################################
+# GETTING HOME TEAM PITCHING STATS
+#####################################################################
+CREATE OR REPLACE TABLE home_pitching_stats
+SELECT
+    EXTRACT(YEAR FROM g.local_date) AS year,
+    g.game_id,
+    g.home_team_id AS home_team,
+    CASE
+        WHEN tpc.Strikeout = 0 THEN 0
+        ELSE tpc.Hit/tpc.Strikeout
+    END AS home_hso,
+    CASE
+        WHEN tpc.Walk = 0 THEN 0
+        ELSE tpc.Strikeout/tpc.Walk
+    END AS home_k_bb,
+    CASE
+        WHEN (tpc.atBat+tpc.Walk+tpc.Hit_By_Pitch+tpc.Sac_Fly) = 0 THEN 0
+        ELSE (tpc.Hit+tpc.Walk+tpc.Hit_By_Pitch)/(tpc.atBat+tpc.Walk+tpc.Hit_By_Pitch+tpc.Sac_Fly)
+    END AS home_obp
+FROM
+    game g
+JOIN team_pitching_counts tpc
+ON tpc.team_id = g.home_team_id
+GROUP BY
+    g.game_id
+ORDER BY
+    g.game_id, g.home_team_id DESC;
+
+#####################################################################
+# GETTING AWAY TEAM STATS
+#####################################################################
+CREATE OR REPLACE TABLE away_batting_stats
+SELECT
+    EXTRACT(YEAR FROM g.local_date) AS year,
+    g.game_id,
+    g.away_team_id AS away_team,
+    CASE
+        WHEN tbc.atBat = 0 THEN 0
+        ELSE tbc.Hit/tbc.atBat
+    END AS away_batting_avg,
+    CASE
+        WHEN tbc.Hit = 0 THEN 0
+        ELSE tbc.Strikeout/tbc.Hit
+    END AS away_soh,
+    CASE
+        WHEN tbc.Hit = 0 THEN 0
+        ELSE tbc.Home_Run/tbc.Hit
+    END AS away_hrh,
+    CASE
+        WHEN tbc.Strikeout = 0 THEN 0
+        ELSE tbc.Walk/tbc.Strikeout
+    END AS away_bb_k
+FROM
+    game g
+JOIN team_batting_counts tbc
+ON tbc.team_id = g.away_team_id
+GROUP BY
+    g.game_id
+ORDER BY
+    g.game_id, g.away_team_id DESC;
+
+#####################################################################
+# GETTING AWAY TEAM PITCHING STATS
+#####################################################################
+CREATE OR REPLACE TABLE away_pitching_stats
+SELECT
+    EXTRACT(YEAR FROM g.local_date) AS year,
+    g.game_id,
+    g.away_team_id AS away_team,
+    CASE
+        WHEN tpc.Strikeout = 0 THEN 0
+        ELSE tpc.Hit/tpc.Strikeout
+    END AS away_hso,
+    CASE
+        WHEN tpc.Walk = 0 THEN 0
+        ELSE tpc.Strikeout/tpc.Walk
+    END AS away_k_bb,
+    CASE
+        WHEN (tpc.atBat+tpc.Walk+tpc.Hit_By_Pitch+tpc.Sac_Fly) = 0 THEN 0
+        ELSE (tpc.Hit+tpc.Walk+tpc.Hit_By_Pitch)/(tpc.atBat+tpc.Walk+tpc.Hit_By_Pitch+tpc.Sac_Fly)
+    END AS away_obp
+FROM
+    game g
+JOIN team_pitching_counts tpc
+ON tpc.team_id = g.away_team_id
+GROUP BY
+    g.game_id
+ORDER BY
+    g.game_id, g.away_team_id DESC;
```

Comment:
Not totally sure what this error is but I'm seeing an "Aggregate Related Problems" Issue